pysat/tests/test_meta.py

```diff
@@ -20,6 +20,10 @@ def teardown(self):
         del self.testInst
         del self.meta
 
+    @raises(ValueError)
+    def test_setting_nonpandas_metadata(self):
+        self.meta = pysat.Meta(metadata='Not a Panda')
```

Comment:
but I like pandas! Do you think if it tries hard enough it can become a panda? Like pinocchio?

Reply:
🐼